pkgs/development/python-modules/python-ipware/default.nix

```diff
@@ -3,24 +3,30 @@
 , fetchFromGitHub
 , pythonOlder
 , unittestCheckHook
+, setuptools
 }:
 buildPythonPackage rec {
   pname = "python-ipware";
-  version = "0.9.0";
-  format = "setuptools";
+  version = "2.0.0";
```

Comment:
I can't check from here, but are there any dependencies on python-ipware and do they still work after this bump?

Reply:
Paperless seems to be the only tool using it. Also nixpkgs-review runs fine